SKILL.tsv

```diff
@@ -3791,72 +3791,72 @@ SKILL_20150414_003790	Temporarily hide yourself away from enemy's vision. You be
 SKILL_20150414_003791	Distort sense of distance and make attack of allies possible outside the range.
 SKILL_20150414_003792	Show hidden items and enemies within range.
 SKILL_20150414_003793	$Temporarily increases your Critical Chance when attacking enemies from behind. 
-SKILL_20150414_003794	Trick enemies to lower their guards. Evasion of enemies in front decreases.
-SKILL_20150414_003795	Hide on the ground. You can attack but cannot move.
-SKILL_20150414_003796	Steal enemy's trap and magic circle. You may use it alter when you need it.
-SKILL_20150414_003797	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Enemy explodes when killed with this attack and your party's strength temporarily increase.
+SKILL_20150414_003794	Trick enemies to lower their guard. Evasion of enemies in front decreases.
+SKILL_20150414_003795	Hide underground to conceal your appearance. In this state you can attack but cannot move.
+SKILL_20150414_003796	Steal enemy's trap and magic circle. You may use it later when you need it.
+SKILL_20150414_003797	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Enemy explodes when killed with this attack and your party's strength temporarily increases.
 SKILL_20150414_003798	{memo X}{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Throw tear gas on target area. Enemies fall under [Dark].
-SKILL_20150414_003799	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Make continuous attack using sword. Double attack with high rate of critical and has chance of fainting enemy.
```

Comment:
Something like:

Launches an arrow towards a specified location, that on impact bursts into flames that damage the target.